packages/@aws-cdk/aws-ivs/README.md

```diff
@@ -9,8 +9,72 @@
 ---
 <!--END STABILITY BANNER-->
 
+Amazon Interactive Video Service (Amazon IVS) is a managed live streaming
+solution that is quick and easy to set up, and ideal for creating interactive
+video experiences. Send your live streams to Amazon IVS using streaming software
+and the service does everything you need to make low-latency live video
+available to any viewer around the world, letting you focus on building
+interactive experiences alongside the live video. You can easily customize and
+enhance the audience experience through the Amazon IVS player SDK and timed
+metadata APIs, allowing you to build a more valuable relationship with your
+viewers on your own websites and applications.
+
 This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
 
+### Channels
+
+An Amazon IVS channel stores configuration information related to your live
+stream. You first create a channel and then contribute video to it using the
+channel’s stream key to start your live stream.
+
+You can create a channel
+
+```ts
+const myChannel = new ivs.Channel(this, 'Channel');
```

Comment:
Can you include the import statement, either on its own, or in this first example?

Reply:
I guess I misinterpreted this? https://github.com/aws/aws-cdk/pull/11454#discussion_r525371781

Reply:
a little bit - the i'd prefer if the individual snippets included the `import`
Ideally, we use a `fixture` as described in the [CONTRIBUTING](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#examples) guide - which can also be verified by running the [compile samples](https://github.com/aws/aws-cdk/blob/master/scripts/compile-samples) script.

otherwise, the generated code samples for other languages will be off.

Reply:
I'm not sure I follow, you want me to add the import to all the snippets? Or you want me to do something with a fixture? Or both?

Reply:
@IainCole - the preference is to create the default fixture (examples in `ec2` and `cognito` modules) so that the code generated for other languages is as useful as the typescript samples.